backend/tarsy/repositories/history_repository.py

```
@@ -302,6 +302,57 @@ def get_alert_sessions(
                     "total_items": 0
                 }
             }
+
+    def get_stage_interaction_counts(self, execution_ids: List[str]) -> Dict[str, Dict[str, int]]:
```

Ruff (0.12.2)

Line too long (98 > 88)

(E501)